Gemfile.lock

```diff
@@ -1,55 +1,55 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (6.0.3.2)
-      actionview (= 6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionpack (6.0.3.3)
+      actionview (= 6.0.3.3)
+      activesupport (= 6.0.3.3)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actionview (6.0.3.2)
-      activesupport (= 6.0.3.2)
+    actionview (6.0.3.3)
```

Comment:
### [actionview](https://rubyonrails.org)

**Hotfix** version upgrade :chart_with_upwards_trend::small_red_triangle: 6.0.3.2 → 6.0.3.3

[_[change-log](https://github.com/rails/rails/blob/v6.0.3.3/actionview/CHANGELOG.md), [source-code](https://github.com/rails/rails/tree/v6.0.3.3/actionview)_]

:tada: Patched vulnerabilities:

- [CVE-2020-15169](https://nvd.nist.gov/vuln/detail/CVE-2020-15169)
  **Potential XSS vulnerability in Action View**

  
  URL: https://groups.google.com/g/rubyonrails-security/c/b-C9kSGXYrc

  There is a potential Cross-Site Scripting (XSS) vulnerability in Action View's translation helpers. Views that allow the user to control the default (not found) value of the `t` and `translate` helpers could be susceptible to XSS attacks.  Impact ------  When an HTML-unsafe string is passed as the default for a missing translation key [named `html` or ending in `_html`](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations), the default string is incorrectly marked as HTML-safe and not escaped. Vulnerable code may look like the following examples:  ```erb <%# The welcome_html translation is not defined for the current locale: %> <%= t("welcome_html", default: untrusted_user_controlled_string) %>  <%# Neither the title.html translation nor the missing.html translation is defined for the current locale: %> <%= t("title.html", default: [:"missing.html", untrusted_user_controlled_string]) %> ```  Workarounds ----------- Impacted users who can’t upgrade to a patched Rails version can avoid this issue by manually escaping default translations with the `html_escape` helper (aliased as `h`):  ```erb <%= t("welcome_html", default: h(untrusted_user_controlled_string)) %> ```






<details>
<summary>Commits</summary>

A change of **2** commits. See the full changes on [the compare page](https://github.com/rails/rails/compare/v6.0.3.2...v6.0.3.3).

These are the individual commits:
- (4ca2027) [Fix XSS vulnerability in `translate` helper](https://github.com/rails/rails/commit/4ca2027d70ddbab141300e2b8331d2d775c63154)
- (070d4af) [v6.0.3.3](https://github.com/rails/rails/commit/070d4afacd3e9721b7e3a4634e4d026b5fa2c32c)


</details>